team milestones/notes.md

```diff
@@ -0,0 +1,43 @@
+# 2. Frontend (HTML landing page)
+- [ ] **create sitemap**
+- [ ] create landing page
+
+## parts of the landing page
+- header
+	- **logo**
+	- **site name**
+	- **navbar**
+		- categories
+		- search bar
+		- site pages
+- hero 1
+	- **headline**
+	- hero image (?)
+	- **register button**
+	- **log in button**    ->> we will place this two buttons in hero? Or in header?
```

Comment:
what do you suggest? we can do both i guess. i just think if we put the buttons in a hero it will be the first thing the site visitor will see. but yeah we can also add a sign in button on the navbar